packages/near-api-js/.eslintrc.yml

```diff
@@ -0,0 +1 @@
+extends: '../../.eslintrc.yml'
```

Comment:
If all we are doing is extending the base config, I don't think this is needed, the base config will just be used by default?

Reply:
I think this is useful to establish the convention of thin per package override. It's empty now but I expect it won't be as more packages are added

Reply:
Wouldn't we want consistent rules across the entire project? I can imagine it becoming quite confusing to have different rule sets across different packages?

Reply:
Cool we can let this pattern evolve if we end up needing it. I'm not sure if we'll need different rules per package but agreed that it would be additional overhead. Let's use one for now